### PR TITLE
Fixed BSC#1225150.

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Added a section what to add to let processes finish first.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: references/combustion-configuration.xml [hunk 1]
```diff
@@ -110,6 +110,16 @@ exec &gt; &gt;(exec tee -a /dev/tty0) 2&gt;&amp;1
...
 echo "Configured with combustion" &gt; /etc/issue.d/combustion</screen>
     </section>
+
+    <section xml:id="combustion-script-wait">
+      <title>Waiting for task to complete</title>
+      <para>
+        To ensure that all running processes are completed before the <filename>script</filename>
+        execution finishes, add the following line:
+      </para>
+      <screen>exec 1&gt;&amp;- 2&gt;&amp;-; wait;</screen>
```
Comment (Vogtinator) [inline references/combustion-configuration.xml:120]: This should be part of the example above

File: references/combustion-configuration.xml [hunk 2]
```diff
@@ -110,6 +110,16 @@ exec &gt; &gt;(exec tee -a /dev/tty0) 2&gt;&amp;1
 # Leave a marker
 echo "Configured with combustion" &gt; /etc/issue.d/combustion</screen>
     </section>
+
+    <section xml:id="combustion-script-wait">
+      <title>Waiting for task to complete</title>
+      <para>
+        To ensure that all running processes are completed before the <filename>script</filename>
```
Comment (Vogtinator) [inline references/combustion-configuration.xml:117]: Maybe mention that this is necessary if background processes are started, like for instance by

```
# Redirect output to the console
exec > >(exec tee -a /dev/tty0) 2>&1
```

**Review by dariavladykina — COMMENTED** [review]
LGTM

**Review by Vogtinator — APPROVED** [review]
LGTM. I think there's another example further down that might need it as well

File: references/combustion-configuration.xml [hunk 3]
```diff
@@ -107,14 +107,20 @@ exec &gt; &gt;(exec tee -a /dev/tty0) 2&gt;&amp;1
           
   nm_config # Configure NM in the system
   curl example.com
+
+# Close outputs and wait for tee to finish.
```
Comment (dariavladykina) [inline references/combustion-configuration.xml:111]:
```suggestion
# Close outputs and wait for tee to finish
```

File: references/combustion-configuration.xml [hunk 4]
```diff
@@ -107,14 +107,20 @@ exec &gt; &gt;(exec tee -a /dev/tty0) 2&gt;&amp;1
...
+exec 1&gt;&amp;- 2&gt;&amp;-; wait;
```
Comment (dariavladykina) [inline references/combustion-configuration.xml:112]: Just a heads-up that now this line is duplicated. I hope this is meant to be.

File: references/combustion-configuration.xml [hunk 5]
```diff
@@ -107,14 +107,20 @@ exec &gt; &gt;(exec tee -a /dev/tty0) 2&gt;&amp;1
...
+
 # Leave a marker
 echo "Configured with combustion" &gt; /etc/issue.d/combustion</screen>
     </section>
 
     <section xml:id="combustion-script-wait">
       <title>Waiting for task to complete</title>
```
Comment (dariavladykina) [inline references/combustion-configuration.xml:119]:
```suggestion
      <title>Waiting for the task to complete</title>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see my comments here. Thanks!

**Pull request merged**